### PR TITLE
Update README.md with instructions to install from source

### PR DESCRIPTION
**Pull request by alexfmpe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,11 @@ Or to launch ghcid for `lib/command` project:
 nix-shell -A obeliskEnvs.obelisk-command --run "cd lib/command && ghcid -c 'cabal new-repl'"
 ```
 
+To re-install `ob` from source do
```
Comment (matthewbauer) [inline README.md]: This should also note that ob will use the version found in your ```.ob``` directory. So updating ob here, may still mean your project's ob is old.

Comment (alexfmpe) [reply README.md]: `.ob` ? Do you mean `.obelisk/impl` ?

Comment (matthewbauer) [reply README.md]: Oops, yeah that's what I meant.

File: README.md [hunk 2]
```diff
@@ -60,6 +60,30 @@ Or to launch ghcid for `lib/command` project:
...
+If you want to commit your changes, first push them to your fork of obelisk and then
+
+```shell
+cd ../../
+ob thunk pack .obelisk/impl
+git add .obelisk/impl
+git commit -m "Bump obelisk"
+```
+
+push them to your obelisk repo,
```
[the remark this answers is not in the document]
Comment (3noch) [reply README.md]: How did I miss this... :joy:

**Pull request merged**